src/stream/src/executor/mview/materialize.rs

```diff
@@ -500,28 +485,30 @@ impl<SD: ValueRowSerde> MaterializeCache<SD> {
                 Op::Delete | Op::UpdateDelete => {
                     match conflict_behavior {
                         ConflictBehavior::Overwrite => {
-                            match self.force_get(&key).as_overwrite().unwrap() {
+                            match self.force_get(&key) {
                                 Some(old_row) => {
                                     fixed_changes.delete(key.clone(), old_row.row.clone());
                                 }
                                 None => (), // delete a nonexistent value
                             };
                             update_cache = true;
                         }
-                        ConflictBehavior::IgnoreConflict => (),
+                        ConflictBehavior::IgnoreConflict => {
+                            match self.force_get(&key) {
+                                Some(old_row) => {
+                                    if old_row.row == value {
+                                        fixed_changes.delete(key.clone(), old_row.row.clone());
+                                        update_cache = true;
+                                    }
```

Comment:
Not sure the expected behavior when the old value is not consitent, need think when we use the Ignore behavior